src/service/api.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { getBlockChain } from './api';
+
+describe('getBlockChain', () => {
+  it('should returns a blockchain ', async () => {
+    // given
+    // when
+    const result = await getBlockChain();
+    // then
+    expect(result.difficulty).toEqual(2);
```

Comment:
2 should be in the given and have an appropriate name.
This is a bad pattern commonly name 'magic number' in software engineering